server/app.js

```diff
@@ -56,6 +60,13 @@ if (process.env.NODE_ENV === "production") {
   });
 }
 
+// POST request for uploading to Express server then pushing to S3 Bucket
+app.post('/profilePhotos', upload.single('profilePhoto'), async (req, res) => {
+  const file = req.file;
+  const result = await uploadPhoto(file);
+  res.send('upload complete!')
```

Comment:
Verify that upload was successful before returning a success response, if not, return a failure status.

Reply:
Got it! Have a .catch to resolve the errors.

Reply:
Please handle errors here. Also, wouldn't it be better to send the image URL as a response?